### PR TITLE
UNOMI-535: cleanup, refacto, stabilization, fixes and new tests for S…

### PR DESCRIPTION
**Pull request by jkevan**
…etPropertyAction, PropertyHelper and IncrementPropertyAction

### CODE REVIEW

File: itests/src/test/java/org/apache/unomi/itests/PropertiesUpdateActionIT.java [hunk 1]
```diff
@@ -71,6 +71,7 @@ public void setUp() throws IOException, InterruptedException {
         profile.setProperties(new HashMap<>());
         profile.setProperty("lastName", "Jose"); // property that have a propertyType registered in the system
         profile.setProperty("prop4", "New property 4"); // property that do not have a propertyType registered in the system
+        profile.setProperty("prop4", "New property 4"); // property that do not have a propertyType registered in the system
```
Comment (sergehuber) [inline itests/src/test/java/org/apache/unomi/itests/PropertiesUpdateActionIT.java:74]: This looks like a copy-paste problem ? The line is identical to the previous one ?

**Review by sergehuber — CHANGES_REQUESTED** [review]
Hi Kevan, thanks for all the changes. Looks good to me apart from a small issue that must be a copy-paste error ?

File: itests/src/test/java/org/apache/unomi/itests/PropertiesUpdateActionIT.java (hunk 1 above)
Comment (jkevan) [reply itests/src/test/java/org/apache/unomi/itests/PropertiesUpdateActionIT.java:74]: yes you are right, I will remove this line.
Thx for the review !

**Review by sergehuber — APPROVED** [review]
Thanks for correcting the line so fast. All good to go now. 

Regards,
  Serge...

**Pull request merged**